JSDemos/Demos/PivotGrid/VirtualScrolling/description.md

```diff
@@ -1 +1,5 @@
-The PivotGrid ships with virtual scrolling support. Virtual scrolling improves the performance when displaying large numbers of&nbsp;PivotGrid cells. The PivotGrid in&nbsp;this demo contains approximately 4000&nbsp;cells. To&nbsp;see virtual scrolling in&nbsp;action, scroll the PivotGrid horizontally.
\ No newline at end of file
+If the PivotGrid is bound to a large dataset, you can enable the virtual scrolling mode to optimize data load time and reduce resource consumption. In this mode, the PivotGrid loads only the cells that are in the viewport. When cells leave the viewport, the PivotGrid removes them from the DOM.
+
+To allow users to scroll the PivotGrid virtually, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
```

Comment:
I don't know about the phrase "scroll the control virtually". You are really (actually) scrolling it. I would just stick the term "Virtual Scrolling", which is the name of the demo. But then I don't want to repeat the words "mode" and "virtual" too many times. So maybe something simple like this:

```suggestion
To enable this mode, set the **scrolling**.[mode](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/scrolling/#mode) to _"virtual"_.
```